tools/benchmark/binsize/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "icu_benchmark_binsize"
+version = "0.1.0"
+authors = ["The ICU4X Project Developers"]
+edition = "2018"
+
+include = [
```

Comment:
Suggestion: I don't think these include files are required, as there's no intention of publishing this separately.

Reply:
Correct, removed.